CONTRIBUTING.rst

```diff
@@ -5,28 +5,27 @@ Contributing
 Contributions are welcome and appreciated!
 Every little bit helps, and credit will always be given.
 
-If you are new to ScanCode and want to find easy tickets to work on, check https://github.com/nexB/scancode-toolkit/labels/easy
+If you are new to ScanCode and want to find easy tickets to work on, check `issues <https://github.com/nexB/scancode-toolkit/labels/easy>`_
 
 When contributing to ScanCode (such as code, bugs, documentation, etc.) you
-agree to the Developer Certificate of Origin http://developercertificate.org/
-and the ScanCode license (see the NOTICE file).  The same approach is used
+agree to the Developer `Certificate of Origin <http://developercertificate.org/>`_
+and the ScanCode license (see the `NOTICE <https://github.com/nexB/scancode-toolkit/blob/develop/NOTICE>`_ file).  The same approach is used
 by the Linux Kernel developers and several other projects.
 
 For commits, it is best to simply add a line like this to your commit message,
 with your name and email::
 
     Signed-off-by: Jane Doe <developer@example.com>
 
-Please try to write a good commit message.
-See https://github.com/nexB/aboutcode/wiki/Writing-good-commit-messages for details.
+Please try to write a good commit message, see `good commit message wiki <https://github.com/nexB/aboutcode/wiki/Writing-good-commit-messages>`_ for details.
+For writing good Pull Request messages and comments with markdown, read `this GitHub guide <https://guides.github.com/features/mastering-markdown/>`_ for once
 
 
 
 Feature requests and feedback
 =============================
 
-To send feedback or ask a question file an issue at
-https://github.com/scancode/scancode-toolkit/issues
+To send feedback or ask a question `file an issue <https://github.com/scancode/scancode-toolkit/issues>`_ 
```

Comment:
The url here is not valid, you need nexB/scancode-toolkit/issues

Also, since you're using this  url in multiple places, you should create the url once, and then reuse it multiple times. See https://raw.githubusercontent.com/alisianoi/license-expression/master/README.rst for reference

Reply:
sure

Reply:
A good resource is found [here](http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#embedded-uris-and-aliases) too. Thanks @alisianoi 😄